chapters/ch06.md

```diff
@@ -1,48 +1,48 @@
+Một khi bạn đã xác định được tập phát triển và tập kiểm tra, đội nhóm của bạn sẽ tập trung cải thiện kết quả trên tập phát triển. Do đó tập phát triển nên phản ánh được việc mà bạn muốn cải thiện nhiều nhất: hoạt động tốt trên dữ liệu từ không chỉ hai mà cả bốn khu vực địa lý.
 
 > There is a second problem with having different dev and test set distributions: There is a chance that your team will build something that works well on the dev set, only to find that it does poorly on the test set. I’ve seen this result in much frustration and wasted effort. Avoid letting this happen to you.
 
-Vấn đề thứ hai với việc tập phát triển và tập kiểm tra có phân phối khác nhau: Có khả năng nhóm của bạn xây dựng được một mô hình hoạt động tốt trên tập phát triển nhưng lại kém trên tập kiểm tra. Tôi đã từng thấy việc này (việc có phân phối khác nhau của hai tập dữ liệu) dẫn đến những kết quả đáng thất vọng và sự lãng phí về công sức. Hãy tránh để điều này xảy ra.
+Vấn đề thứ hai đối với sự phân phối của tập phát triển và tập kiểm tra: Có khả năng đội nhóm của bạn xây dựng được một thứ gì đó hoạt động tốt trên tập phát triển nhưng lại kém trên tập kiểm tra. Tôi đã từng thấy điều này  dẫn đến những kết quả đáng thất vọng và lãng phí công sức ra sao . Hãy tránh để nó xảy ra với bạn.
 
 > As an example, suppose your team develops a system that works well on the dev set but not the test set. If your dev and test sets had come from the same distribution, then you would have a very clear diagnosis of what went wrong: You have overfit the dev set. The obvious cure is to get more dev set data.
 
-Ví dụ, nhóm của bạn phát triển một hệ thống hoạt động tốt trên tập phát triển nhưng kém trên tập kiểm tra. Nếu tập phát triển và tập kiểm tra có cùng một phân phối thì bạn có thể xác định ngay vấn đề: Mô hình của bạn đã overfit tập phát triển. Cách xử lý hiển nhiên nhất đó là bổ sung thêm dữ liệu cho tập phát triển.
+Ví dụ như, đội nhóm của bạn xây dựng một hệ thống hoạt động tốt trên tập phát triển nhưng kém trên tập kiểm tra. Nếu tập phát triển và tập kiểm tra có cùng một tập phân phối thì bạn có thể chuẩn đoán chính xác chỗ nào sai: Tập phát triển của bạn đã bị thiếu. Hiển nhiên là bạn cần bổ sung thêm dữ liệu cho tập phát triển.
 
 > But if the dev and test sets come from different distributions, then your options are less clear. Several things could have gone wrong:
 
-Nhưng nếu tập phát triển và tập kiểm tra được tạo từ các phân phối khác nhau thì việc xác định vấn đề sẽ phức tạp hơn. Rất nhiều vấn đề có thể xảy ra:   
-
+Nhưng nếu tập phát triển và tập kiểm tra được tạo từ các tập phân phối khác nhau thì việc xác định vấn đề sẽ phức tạp hơn. Rất nhiều thứ có thể sai:   
+  
 > 1. You had overfit to the dev set.
 
-1. Bạn đã overfit tập phát triển.
+1. Tập phát triển của bạn bị thiếu dữ liệu.
```

Comment:
* Bạn vui lòng giữ lại từ "overfit", thiếu dữ liệu chỉ là một nhân tốt làm nên overfit, overfit còn bao hàm nhiều yếu tố khác nữa.

Reply:
Cám ơn bạn đã nói rõ hơn về overfit. Nếu giữ lại thuật ngữ này mình nghĩ nên dịch theo hướng danh từ chứ động từ thì không ổn, người đọc sẽ khó hiểu. Hoặc là bạn tạo issue để dịch thuật ngữ đó ra tiếng Việt.

Reply:
Ở đây mình dịch là "thiếu dữ liệu" bởi vì tá giả đang nói về vấn đề dữ liệu, và do đó dev set cần được bổ sung thêm dữ liệu để chạy đúng

Reply:
Một lần nữa, mình rất trân trọng những đề xuất của bạn. Nhưng việc dịch không chỉ đơn giản là viết theo cách hiểu của mình mà cần có thêm kiến thức chuyên ngành nữa. "Overfit" là một thuật ngữ chính rất quan trọng và có cả một mảng lớn nghiên cứu về nó. Đây là một bài mình viết về overfitting, nó không chỉ đơn giản là thiếu dữ liệu như bạn nghĩ:
https://machinelearningcoban.com/2017/03/04/overfitting/

> Ở đây mình dịch là "thiếu dữ liệu" bởi vì tá giả đang nói về vấn đề dữ liệu
Bạn có thể trích lại đoạn tác giả nói về vấn đề này được không? Mình đã tìm nhưng chưa thấy.

Reply:
Dev set hay test set chính là các tập dữ liệu. Về thuật ngữ overfit thì ở trên mình đã đồng ý nếu bạn muốn không dịch, nhưng sẽ nghĩ cách dùng theo danh từ chứ dịch như "nhóm bạn đã overfit tập phát triển" mình thấy chưa ổn. Mình cũng xuất tạo issue để thảo luận về thuật ngữ này dịch ra sao, ý bạn thế nào?
Thông tin thêm với bạn tài liệu mình đã đọc về overfitting
https://elitedatascience.com/overfitting-in-machine-learning

Reply:
"You had overfit to the dev set" dịch là "Tập phát triển của bạn bị tình trạng overfit" thì bạn thấy sao?